subprojects/ide/src/main/java/org/gradle/plugins/ide/internal/tooling/EclipseModelBuilder.java

```diff
@@ -197,14 +199,32 @@ private void populate(Project project) {
         }
 
         DefaultEclipseProject eclipseProject = findEclipseProject(project);
+
         eclipseProject.setClasspath(externalDependencies);
         eclipseProject.setProjectDependencies(projectDependencies);
         eclipseProject.setSourceDirectories(sourceDirectories);
         eclipseProject.setClasspathContainers(classpathContainers);
         eclipseProject.setOutputLocation(outputLocation != null ? outputLocation : new DefaultEclipseOutputLocation("bin"));
 
+        // Create as a holder object.
```

Comment:
Was this included in the "remove all the inline comments" request? I see comments are fairly sparse around here (self-documenting code FTW) but I think that anything that might be "surprising" should be explained.

Reply:
Yes, this was included as well, as it only repeats what the code is saying.